src/App.js

```diff
@@ -9,8 +9,6 @@ class App extends Component {
     super();
     this.state = {
       movies: [],
```

Comment:
Great job removing that part and following Kayla's (instructor) hint to manage pages using router